src/test/js/cookies.spec.js

```diff
@@ -165,4 +177,47 @@ describe('cookies', function() {
             expect(cookieNoticeDialogMock.close).toHaveBeenCalled();
         });
     });
+
+    describe('cookiesStorage -', function() {
+
+        beforeEach(inject(function (cookiesStorage) {
+            sut = cookiesStorage;
+            spy = jasmine.createSpyObj('spy', ['acceptCookies', 'rejectCookies', 'getCookieStorageValue']);
+        }));
+
+        it('Should get the cookiestoragevalue', function() {
+            var result = sut.getCookieStorageValue();
+            expect(result).toBeUndefined();
+
+            sut.acceptCookies();
+            result = sut.getCookieStorageValue();
+            expect(result).toBeTruthy();
+
+            sut.rejectCookies();
+            result = sut.getCookieStorageValue();
+            expect(result).toBeFalsy();
```

Comment:
Ik maak me hier een beetje zorgen. Als de cookiesAccepted value al aanwezig is in de storage, dan kan het zijn dat er geen boolean wordt teruggegeven maar "true" of "false" (als string). Als dit het geval is, dan zal zowel "true" als "false" als truthy aanzien worden.